src/chains/ethereum/ethereum/src/provider.ts

```diff
@@ -91,14 +91,14 @@ function hookEventSystem(
 }
 
 type Primitives = string | number | null | undefined | symbol | bigint;
-type Clean<X> = X extends Primitives
+export type Externalize<X> = X extends Primitives
```

Comment:
Some comments explaining these types while they're fresh would be great :sweat_smile:

Reply:
Done